src/lib/assets/mui_components/AccessManagement.tsx

```diff
@@ -2,38 +2,15 @@ import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon'
 import * as React from 'react'
 
 const AccessManagement = (props: SvgIconProps) => {
-  const { fill, style } = props
-
   return (
-    <SvgIcon style={style}>
-      <path
-        d="M10.959 10.8524C11.0694 12.9636 11.7494 14.0782 12.0009 14.0782C12.2523 14.0782 12.9331 12.9636 13.0427 10.8524H10.959Z"
-        fill={fill}
-      />
-      <path
-        d="M9.7823 9.71382C9.83136 8.67917 10.0123 7.65424 10.3381 6.85657C9.19045 7.38735 8.35251 8.44816 8.15625 9.71382H9.7823Z"
-        fill={fill}
-      />
-      <path
-        d="M14.2172 9.71382H15.8432C15.6469 8.44816 14.809 7.38735 13.6621 6.85657C13.9872 7.65424 14.1681 8.67917 14.2172 9.71382Z"
-        fill={fill}
-      />
-      <path
-        d="M12.0009 6.48877C11.7494 6.48877 11.0686 7.60266 10.959 9.71383H13.0435C12.9331 7.60266 12.2523 6.48877 12.0009 6.48877Z"
-        fill={fill}
-      />
-      <path
-        d="M9.7823 10.8524H8.15625C8.35251 12.1188 9.19045 13.1789 10.3381 13.7104C10.0131 12.9127 9.83136 11.8871 9.7823 10.8524Z"
-        fill={fill}
-      />
-      <path
-        d="M17.6643 2.8667H6.33489C4.49265 2.8667 3 4.32224 3 6.11867V19.6147C3 21.4097 4.49265 22.8667 6.33489 22.8667H17.6643C19.5066 22.8667 21 21.4104 21 19.6147V6.11867C20.9992 4.32224 19.5066 2.8667 17.6643 2.8667ZM16.5504 18.6309C16.5504 19.3493 15.9532 19.9317 15.2164 19.9317H8.78278C8.04604 19.9317 7.44883 19.3493 7.44883 18.6309V18.4679C7.44883 17.7503 8.04604 17.1679 8.78278 17.1679H15.2164C15.9532 17.1679 16.5504 17.7503 16.5504 18.4679V18.6309ZM11.9996 15.2167C9.21057 15.2167 6.94131 13.0039 6.94131 10.2834C6.94131 7.56299 9.21057 5.35016 11.9996 5.35016C14.7887 5.35016 17.0579 7.56299 17.0579 10.2834C17.0579 13.0039 14.7887 15.2167 11.9996 15.2167Z"
-        fill={fill}
-      />
-      <path
-        d="M14.216 10.8524C14.1669 11.8878 13.986 12.9127 13.6602 13.7104C14.8071 13.1789 15.645 12.1188 15.8413 10.8524H14.216Z"
-        fill={fill}
-      />
+    <SvgIcon {...props}>
```

Comment:
In all of these SVGs (except for Kinomics and Proteomics, which are exceptions), pass all props to SvgIcon and drop the `fill` props passed to each `path` tag. SvgIcon will take care of the coloring!